doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -3431,7 +3431,7 @@ \subsubsection{Inputs}\label{inputs-9-014}
 
 \paragraph{Field: Condenser Outlet Node Name}\label{field-condenser-outlet-node-name-8}
 
-This required alpha field contains the identifying name for the chiller condenser side outlet node.
+This required alpha field contains the identifying name for the chiller condenser side outlet node. If the chiller is AirCooled, a node name must be specified but is not used.
```

Comment:
Or one step further - follow the example of Chiller:Electric:EIR where both the condenser inlet and outlet nodes are not required for air cooled and evap cooled.